### PR TITLE
Enhance postgre sql setup script and documentation for various distros 7636

### PR DESCRIPTION
**Pull request by dostavic**

### CODE REVIEW

File: packages/twenty-front/src/pages/settings/data-model/constants/FieldTypeIcons.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { FieldMetadataType } from '~/generated-metadata/graphql';
```
Comment (charlesBochet) [inline packages/twenty-front/src/pages/settings/data-model/constants/FieldTypeIcons.ts:1]: @dostavic what is this change about?

**Review by charlesBochet — APPROVED** [review]
Thank you @dostavic

I'm not sure about the FieldTypeIcon.ts file change? Could you remove this change?

Regarding the provision-postres-linux.sh, looks great! I don't have linux setup on my side so I will trust you on this one :)

**Comment by dostavic** [comment]
Thank you, @charlesBochet!

Regarding the file change, that was my mistake 😊. I've removed it now

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
Thank you for improving the experience on Linux. If you see anything else, do not hesitate!

**Pull request merged**